src/Generating/Trees.cpp

```diff
@@ -58,7 +59,7 @@ static const Vector3d & pickBranchDirection(const cNoise a_Noise, Vector3i a_Blo
 		}
 	};
 
-	size_t index = static_cast<size_t>(a_Noise.IntNoise3DInt(a_BlockPos.x, a_BlockPos.y + a_Seq, a_BlockPos.z)) % directions.size();
+	size_t index = static_cast<size_t>(a_Noise.IntNoise3DInt(a_BlockPos.x * a_Seq, a_BlockPos.y * a_Seq, a_BlockPos.z * a_Seq)) % directions.size();
```

Comment:
Are you concerned about overflow here? How large does `a_Seq` get?

Reply:
No i'm not

i am now takinng  ```a_Noise.IntNoise3DInt(a_BlockPos * a_Seq)``` from line 1314 now - this is old code which worked